ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/Balance.kt

```diff
@@ -24,6 +30,20 @@ private fun StyledBalanceComposablePreview() {
             Column {
                 StyledBalance(
                     balanceParts = ZecAmountTriple(main = "1,234.56789012"),
+                    isHideBalances = false,
+                    textStyles =
```

Comment:
improvement for future - do not use pair, use immutable object instead

Reply:
It's a good idea. I just filed a follow-up for this #1499.